tests/integration/defs/disaggregated/test_auto_scaling.py

```
@@ -246,19 +227,38 @@ async def wrapper(*args, **kwargs):
     return decorator
 
 
-@periodic_check(timeout=300, interval=3)
-async def wait_for_disagg_server_ready(port):
+async def _wait_for_disagg_server_status(port,
+                                         ready=True,
+                                         min_ctx_workers=-1,
+                                         min_gen_workers=-1):
     info_resp = requests.get(f"http://localhost:{port}/cluster_info")
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)